herd/semExtra.ml

```diff
@@ -187,11 +187,10 @@ type concrete =
   type barrier = A.barrier
   type pp_barrier = { barrier:barrier ; pp:string; }
 
-  include
-    IFetchTrait.S
+include
+  IFetchTrait.S
     with type ifetch_instruction := instruction
-     and type ifetch_reg := A.reg
-
+    and type ifetch_reg := A.reg
```

Comment:
are these unintended changes?

Reply:
They are, thanks.

Reply:
Well looking again  I like the new indentation better..